bin/out

```diff
@@ -44,6 +44,39 @@ start_job() {
     # TODO make sure job is started
 }
 
+delete_resource() {
+    RESOURCE_TYPE=$1
+    RESOURCE=$2
+
+    [ -n "$RESOURCE_TYPE" ] || exit 1
+    [ -n "$RESOURCE" ] || exit 1
+
+    $KUBECTL delete "$RESOURCE_TYPE" "$RESOURCE" || true
+}
+
+create_resource() {
+    RESOURCE_FILE=$1
+    IMG=$2
+ 
+    [ -n "$RESOURCE_FILE" ] || exit 1
+    [ -n "$IMG" ] || exit 1
+
+    
+    sed -i "s|image:.*|image: $IMG|g" $RESOURCE_FILE
+
+    $KUBECTL create -f "$RESOURCE_FILE"
+}
+
+rise_from_the_ashes() {
+    RESOURCE_TYPE=$1
+    RESOURCE=$2
+    RESOURCE_FILE=$3
+    IMG=$4
+ 
+    delete_resource "$RESOURCE_TYPE" "$RESOURCE"
+    create_resource "$RESOURCE_FILE" "$IMG"    
+}
+
```

Comment:
haha, this is cute, but can we call this `recreate` or `replace` and add an alias to it that is called `rise_from_the_ashes`? that way you get to stay cute, but it's still explicit?